src/app/drag-and-drop/drag-and-drop.demo.html

```diff
@@ -0,0 +1,179 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+<h2>Drag and Drop</h2>
+
+<div class="clr-example">
+    <h4>Basic Draggable</h4>
+    <div class="row">
+        <div class="col-xs-12">
+            <p>
+                Use <code class="clr-code">clrDraggable</code> directive to make an element or component draggable.
+            </p>
+            <button class="btn btn-primary" clrDraggable>Draggable Button</button>
+            <button class="btn btn-success" clrDraggable>Draggable Button</button>
+            <button class="btn btn-warning" clrDraggable>Draggable Button</button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 1</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 2</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 3</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="clr-example">
+    <h4>Basic Draggable</h4>
+    <div class="row">
+        <div class="col-xs-12">
+            <p>
+                Use <code class="clr-code">clrDragAndDropGroup</code> directive to make particular draggables interact
+                with particular droppables.
+            </p>
+            <button class="btn btn-primary" clrDraggable clrDragAndDropGroup="dnd-1">Draggable Button</button>
```

Comment:
Do you think we can just name this `clrDropGroup` at both places? with `clrDraggable` and `clrDroppable`?

Reply:
I agree `clrDropGroup` is better than `clrDragAndDropGoup`. Also, we could use `clrDroppableAccept`? Because I see that it's convention to use the term `accept` in other DnD libraries. What do you think? @youdz could you also, chime in on this?

Reply:
Also, If we use `clrDroppableAccept` on `clrDroppable`, we need to use some other input name for clrDraggable.

Reply:
How about `clrDropLink` on `clrDraggable` and `clrDropLinkGroup` on `clrDroppable`?